docs/guides/api-hooks.md

```diff
@@ -114,6 +114,7 @@ on day-of-month 1) or `cron(5 4 * * sun)` (at 04:05 on Sunday). See example belo
 module.exports = function registerHook() {
 	return {
 		'cron(*/15 * * * *)': async function () {
+			const axios = require('axios');
```

Comment:
Lets move it to the top of the file (outside the register function). That way, it's only loaded once (otherwise the require happens every time the hook fires)

Reply:
Makes sense 👍